### DISCUSSION TITLE
Possible to use the auto naming and exclude first and last chapters?

### DISCUSSION THREAD

**Discussion by haptiqblack** [yermak/AudioBookConverter#290] [Ideas]
Was wondering if it is possible to exclude first and last in the auto naming convention as sometimes the book will have a prologue and epilogue and skipping the first one can be very helpful, so I don't have to manually name them all.

**Comment by yermak** [comment]
That's an interesting idea. I can't promisse it will be quickly added, but definitely would consider it.
Probably one of way to do this - "edit chapter" and set flag skip numbering, as there could be more than 1 prologue - i've seen a few
